setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name='hsl_builder',
-    version='0.0.1',
+    version='0.a1',
```

Comment:
Can we try Semantic versioning? https://semver.org/
Example - `0.0.1-alpha`

Reply:
pip uses [PEP 440](https://packaging.python.org/guides/distributing-packages-using-setuptools/#choosing-a-versioning-scheme).
we can upload using `0.0.1-alpha` version tag however pypi updates it to `0.0.1a0`
for reference check [this](https://test.pypi.org/project/hsl-builder/0.0.1a0/) which was uploaded with version set as `0.0.1-alpha`